### PR TITLE
homework 1 done

### PR DESCRIPTION
**Pull request by VShutkina**

### CODE REVIEW

File: src/test/java/hw1/common/AbstractBaseTest.java [hunk 1]
```diff
@@ -0,0 +1,21 @@
+package hw1.common;
+
+import com.epam.tat.module4.Calculator;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class AbstractBaseTest {
```
Comment (khda91) [inline src/test/java/hw1/common/AbstractBaseTest.java]: Please format code according Java Code Convention. Please check all project :)

File: src/test/java/hw1/tests/CalculatorDivisionTest.java [hunk 2]
```diff
@@ -0,0 +1,16 @@
+package hw1.tests;
+
+import hw1.common.AbstractBaseTest;
+import hw1.common.CalculatorDataProvider;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CalculatorDivisionTest extends AbstractBaseTest {
+    @Test(dataProviderClass = CalculatorDataProvider.class,
```
Comment (khda91) [inline src/test/java/hw1/tests/CalculatorDivisionTest.java:11]: Here should be empty line between class name and annotation. Please follow Java Code Convention

File: src/test/java/hw1/tests/CalculatorDivisionTest.java [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+            dataProvider = "DivisionTestDataProvider")
+    public void divisionTest(int a, int b, int expected) {
+        System.out.println("Division test");
+        long actual = calculator.div(a, b);
+        Assert.assertEquals(actual, expected);
```
Comment (khda91) [inline src/test/java/hw1/tests/CalculatorDivisionTest.java]: It is better to use `static import` for the `Assert` methods

**Review by khda91 — COMMENTED** [review]
Please fix all comments

Project does not compile. The error: 
```
EPRUPETW2288:ValentinaShutkina dmitry_khodakovsky$ mvn clean test
[INFO] Scanning for projects...
[INFO] 
[INFO] -----------< com.epam.training.center:test-auto-autumn-2019 >-----------
[INFO] Building test-auto-autumn-2019 1.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
[WARNING] The POM for com.epam.calculator:calculator:jar:1.0 is missing, no dependency information available
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  0.411 s
[INFO] Finished at: 2019-11-18T16:05:27+03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project test-auto-autumn-2019: Could not resolve dependencies for project com.epam.training.center:test-auto-autumn-2019:jar:1.0-SNAPSHOT: Failure to find com.epam.calculator:calculator:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Please fix this error

File: src/test/java/hw1/tests/CalculatorDivisionTest.java (hunk 3 above)
Comment (VShutkina) [reply src/test/java/hw1/tests/CalculatorDivisionTest.java]: fixed

File: src/test/java/hw1/tests/CalculatorDivisionTest.java (hunk 2 above)
Comment (VShutkina) [reply src/test/java/hw1/tests/CalculatorDivisionTest.java:11]: fixed

File: src/test/java/hw1/common/AbstractBaseTest.java (hunk 1 above)
Comment (VShutkina) [reply src/test/java/hw1/common/AbstractBaseTest.java]: fixed

**Comment by VShutkina** [comment]
> Please fix all comments
> 
> Project does not compile. The error:
> 
> ```
> EPRUPETW2288:ValentinaShutkina dmitry_khodakovsky$ mvn clean test
> [INFO] Scanning for projects...
> [INFO] 
> [INFO] -----------< com.epam.training.center:test-auto-autumn-2019 >-----------
> [INFO] Building test-auto-autumn-2019 1.0-SNAPSHOT
> [INFO] --------------------------------[ jar ]---------------------------------
> [WARNING] The POM for com.epam.calculator:calculator:jar:1.0 is missing, no dependency information available
> [INFO] ------------------------------------------------------------------------
> [INFO] BUILD FAILURE
> [INFO] ------------------------------------------------------------------------
> [INFO] Total time:  0.411 s
> [INFO] Finished at: 2019-11-18T16:05:27+03:00
> [INFO] ------------------------------------------------------------------------
> [ERROR] Failed to execute goal on project test-auto-autumn-2019: Could not resolve dependencies for project com.epam.training.center:test-auto-autumn-2019:jar:1.0-SNAPSHOT: Failure to find com.epam.calculator:calculator:jar:1.0 in https://repo.maven.apache.org/maven2 was cached in the local repository, resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]
> [ERROR] 
> [ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
> [ERROR] Re-run Maven using the -X switch to enable full debug logging.
> [ERROR] 
> [ERROR] For more information about the errors and possible solutions, please read the following articles:
> [ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
> ```
> 
> Please fix this error

fixed

**Review by khda91 — APPROVED** [review]
Approved. Could be merged to the master branch

**Pull request merged**